app/scripts/modules/amazon/src/loadBalancer/details/loadBalancerDetails.html

```diff
@@ -24,11 +24,17 @@ <h4 class="text-center">
       <h3 select-on-dbl-click>
         {{ctrl.loadBalancer.name}}
         <render-if-feature feature="entityTags">
-          <entity-ui-tags component="ctrl.loadBalancer"
-                          application="ctrl.application"
-                          entity-type="loadBalancer"
-                          page-location="details"
-                          on-update="ctrl.application.loadBalancers.refresh()"></entity-ui-tags>
+
+          <entity-notifications-wrapper
```

Comment:
Fine with me.  I thought it would be more consistent to name the wrapper components (the ones that allow arbitrary `@` or `&` angular bindings) the same way.